src/Doctrine/Common/Filter/RangeFilterTrait.php

```diff
@@ -124,18 +125,18 @@ private function normalizeBetweenValues(array $values): ?array
     /**
      * Normalize the value.
      *
-     * @return int|float|null
+     * @return int|float|string | null
      */
     private function normalizeValue(string $value, string $operator)
     {
-        if (!is_numeric($value)) {
+        if (!is_numeric($value) && !UuidV6::isValid($value)) {
             $this->getLogger()->notice('Invalid filter ignored', [
                 'exception' => new InvalidArgumentException(sprintf('Invalid value for "[%s]", expected number', $operator)),
             ]);
 
             return null;
         }
 
-        return $value + 0; // coerce $value to the right type.
```

Comment:
You shouldn't change this code.

Reply:
You can't keep it with uuid's though, I can add a clause to differentiate though